website/thaliawebsite/settings.py

```diff
@@ -571,7 +575,7 @@ def from_env(
         "thaliawebsite.api.throttling.AnonRateThrottle",
         "thaliawebsite.api.throttling.UserRateThrottle",
     ],
-    "DEFAULT_THROTTLE_RATES": {"anon": "100/day", "user": "1000/day"},
+    "DEFAULT_THROTTLE_RATES": {"anon": "100/day", "user": "10000/day"},
```

Comment:
Maybe it would be better to make it like 20/min? This means that it is basically unlimited if you want to do 10000 calls in a minute.

```suggestion
    "DEFAULT_THROTTLE_RATES": {"anon": "100/day", "user": "20/min"},
```